solc/CommandLineInterface.cpp

```diff
@@ -484,23 +491,33 @@ bool CommandLineInterface::parseLibraryOption(string const& _input)
 	return true;
 }
 
+
+//
+//HERE - Error out instead of skipping non-existent source files #3037
+//
 void CommandLineInterface::createFile(string const& _fileName, string const& _data)
```

Comment:
This is the routine that writes output files, you should not need to touch this.

Reply:
@chriseth Yeah, I plan on removing that comment as it was a placeholder for me to simply orient myself in the code.